app/controllers/password_resets_controller.rb

```diff
@@ -31,10 +33,10 @@ def update
 
   private
   def password_validate
-    if user_params[:password].empty?
-      @user.errors.add :password, t("reset_pass.empty")
-      render :edit
-    end
+    return unless user_params[:password].empty?
```

Comment:
```suggestion
    return if user_params[:password].present?
```
Sao ko dùng ri cho khỏe, cái thèn unless hại não quá cu